std/conv.d

```diff
@@ -2228,7 +2258,7 @@ body
     immutable uint beyond = (radix < 10 ? '0' : 'a'-10) + radix;
 
     Target v = 0;
-    size_t atStart = true;
+    bool atStart = true;
```

Comment:
Old code issue, `atStart` is used as a bool, there's no need (and it's wasteful) to declare it as `size_t`. Ironically enough, this came from a PR that ostensibly wanted to replace `int` variables with `bool` where they are only used as bools. Apparently this line was copied from the previous code but the submitter forgot to change the type to `bool` before committing.